slack_sdk/scim/v1/async_client.py

```diff
@@ -321,16 +320,13 @@ async def _perform_http_request(
                 "proxy": self.proxy,
             }
             async with session.request(http_verb, url, **request_kwargs) as res:
-                response_body = {}
+                response_body: str = ""
                 try:
                     response_body = await res.text()
                 except aiohttp.ContentTypeError:
                     self.logger.debug(
                         f"No response data returned from the following API call: {url}."
                     )
-                except json.decoder.JSONDecodeError as e:
```

Comment:
As this error is never raised, I've removed the except clause here. This error may be raised only when calling `await res.json()` in this part.